godot/src/main.gd

```diff
@@ -7,6 +7,8 @@ func _ready():
 
 
 func start():
+	get_tree().quit_on_go_back = false
+
```

Comment:
I reviewed and you also have it in the project settings

Application > Config > Quit On Go Back